mise.toml

```diff
@@ -351,7 +351,7 @@ echo '###############################################'
 echo
 mise run test:integration:lang:golang
 mise run test:integration:lang:python
-mise run test:integration:lang:elixir
+# mise run test:integration:lang:elixir
```

Comment:
I think it might help to add a comment why this is commented out here in case it takes a while to get back to this issue but I'm hoping that won't be the case